tools-local/doc.tasks/GenerateMarkdownDoc.cs

```diff
@@ -94,15 +96,31 @@ private void WriteSectionProjecs(ITaskItem section, StreamWriter file)
             file.WriteLine($"Projects in section: {projectFiles.Length}\n");
                     
             Array.Sort(projectFiles, (a, b) => a.CompareTo(b));
+            Array.ForEach(projectFiles, pf => pf = RelativizePath(pf));
```

Comment:
Is this modifying the array?

Reply:
Yes

Reply:
Nope - I was wrong. This is not actually doing anything useful.. removing it.